### PR TITLE
#416 tests implemented

### PR DESCRIPTION
**Pull request by ItamarRocha**
- Alterations in file runtests.jl
- Creation of file test_branching_file.jl

### CODE REVIEW

File: test/test_branching_file.jl [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+                    if pieceofdata[1] == 'n'
+                        push!(existing_nodes, Int(pieceofdata[2]))
+                    end
+                end
+            end
+        end
+        return length(existing_nodes)
+    end
+    
+    @testset "play gap" begin
```
Comment (guimarqu) [inline test/test_branching_file.jl:19]: Can you move the test in the `issues_tests.jl` file and rename the test set ?

File: test/test_branching_file.jl [hunk 2]
```diff
@@ -0,0 +1,43 @@
+function branching_file_completion()
+    function get_number_of_nodes_in_branching_tree_file(filename::String)
+        filepath = string(@__DIR__ , "/", filename)
+        
+        existing_nodes = Set()
+        
+        open(filepath) do file
+            for line in eachline(file)
+                for pieceofdata in split(line)
+                    if pieceofdata[1] == 'n'
```
Comment (guimarqu) [inline test/test_branching_file.jl:10]: Can you use a regex here ? It will be easier to update the test.

File: test/test_branching_file.jl [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        model, x, dec = CLD.GeneralizedAssignment.model(data, coluna)
+        BD.objectiveprimalbound!(model, 100)
+        BD.objectivedualbound!(model, 0)
+
+        JuMP.optimize!(model)
+
+        @test MOI.get(model, MOI.NodeCount()) == get_number_of_nodes_in_branching_tree_file("playgap.dot")
+        @test JuMP.objective_value(model) ≈ 75.0
+        @test JuMP.termination_status(model) == MOI.OPTIMAL
+        @test CLD.GeneralizedAssignment.print_and_check_sol(data, model, x)
```
Comment (guimarqu) [inline test/test_branching_file.jl:39]: We can keep the 3 first tests only

File: test/test_branching_file.jl (hunk 1 above)
Comment (ItamarRocha) [reply test/test_branching_file.jl:19]: Ok

File: test/test_branching_file.jl (hunk 3 above)
Comment (ItamarRocha) [reply test/test_branching_file.jl:39]: Done

File: test/test_branching_file.jl (hunk 2 above)
Comment (ItamarRocha) [reply test/test_branching_file.jl:10]: Ok

**Comment by ItamarRocha** [comment]
@guimarqu

**Review by guimarqu — APPROVED** [review]

**Pull request merged**